client/src/pages/reports/AttendeesTable.js

```diff
@@ -5,6 +5,8 @@ import React from "react"
 import { Button, Label, Radio, Table } from "react-bootstrap"
 import REMOVE_ICON from "resources/delete.png"
 import "./AttendeesTable.css"
+import PlanningConflictForPerson from "components/PlanningConflictForPerson"
+import Report from "models/Report"
```

Comment:
Please properly order the imports (alphabetically on imported package if there's no command for it in your IDE).